ReactiveFeedback/Feedback.swift

```diff
@@ -12,10 +13,23 @@ public struct Feedback<State, Event> {
     /// - parameters:
     ///   - events: The transform which derives a `Signal` of events from the
     ///             latest state.
-    public init(events: @escaping (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>) {
+    public init(events: @escaping (Scheduler, Signal<(Event?, State), NoError>) -> Signal<Event, NoError>, _ dummy: Void = ()) {
```

Comment:
What is `dummy`

Reply:
This is a workaround technique to disambiguate 2 `.init { ... }`s by prohibiting new one from using trailing closure, i.e. `.init(events: { ... })` will only work.

And default value is automatically provided, so `dummy` will be ignorable to users (though signature looks ugly).